src/main/application/BorosTcf.js

```diff
@@ -20,13 +22,15 @@ class BorosTcf {
     loadUserConsentUseCase = inject(LoadUserConsentUseCase),
     saveUserConsentUseCase = inject(SaveUserConsentUseCase),
     getTCDataUseCase = inject(GetTCDataUseCase),
-    changeUiVisibleUseCase = inject(ChangeUiVisibleUseCase)
+    changeUiVisibleUseCase = inject(ChangeUiVisibleUseCase),
+    statusRepository = inject(StatusRepository)
   } = {}) {
     this._getVendorListUseCase = getVendorListUseCase
     this._loadUserConsentUseCase = loadUserConsentUseCase
     this._saveUserConsentUseCase = saveUserConsentUseCase
     this._getTCDataUseCase = getTCDataUseCase
     this._changeUiVisibleUseCase = changeUiVisibleUseCase
+    statusRepository.getStatus().cmpStatus = Status.CMPSTATUS_LOADED
```

Comment:
NBC
tech debt: why if we have a repository we're mutating the Status here? If we do not update the status via the repository, then the Status should be a "singleton" shared across the the use cases without any repository to access it